packages/calcite-components/src/components/meter/meter.scss

```diff
@@ -1,57 +1,110 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-meter-background-color: Specifies the background color of the component.
+ * @prop --calcite-meter-border-color: Specifies the border color of the component and displayed step lines.
+ * @prop --calcite-meter-shadow: Specifies the box shadow of the component.
+ * @prop --calcite-meter-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-meter-fill-color: Specifies the color of the component's fill.
+ * @prop --calcite-meter-range-text-color: Specifies the color of the component's range labels.
+ * @prop --calcite-meter-value-text-color: Specifies the color of the component's value label.
+ *
+ */
+
+/**
+  * Local props
+  * These properties are intended for internal component use only. It is not recommended that these properties be overwritten.
+  *
+  * --calcite-internal-meter-fill-color
+  * --calcite-internal-meter-background-color
+  * --calcite-internal-meter-space
+  * --calcite-internal-meter-height
+  * --calcite-internal-meter-font-size
+*/
+
 @include base-component();
 @include disabled();
 
 :host {
   @apply flex;
-  --calcite-meter-space-internal: theme("spacing[0.5]");
-  --calcite-meter-height-internal: theme("spacing.4");
-  --calcite-meter-font-size-internal: var(--calcite-font-size--1);
+  --calcite-internal-meter-space: var(--calcite-spacing-base);
+  --calcite-internal-meter-height: var(--calcite-spacing-lg);
+  --calcite-internal-meter-font-size: var(--calcite-font-size--1);
+  --calcite-meter-fill-color: var(--calcite-meter-fill-color, var(--calcite-color-brand));
```

Comment:
I think this line can be dropped. All internal usages of `--calcite-meter-fill-color` have a fallback.

Reply:
This doesn't work without this line here - we use the internal definition throughout below.